NameLayerMain/src/vg/civcraft/mc/namelayer/permission/GroupPermission.java

```diff
@@ -69,14 +70,20 @@ public String listPermsforPlayerType(PlayerType type){
 	 * @param permType- The PermissionType.
 	 * @return Returns false if the PlayerType already has the permission.
 	 */
-	public boolean addPermission(PlayerType pType, PermissionType permType){
+	public boolean addPermission(PlayerType ptype, PermissionType permtype){
```

Comment:
Similar complaint here; the previous standard / published interfaces where all synchronous requests. Here they are now forced to by async. This is not good.